tensorflow/lite/delegates/nnapi/java/src/main/native/nnapi_delegate_jni.h

```diff
@@ -0,0 +1,37 @@
+/* Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+#ifndef TENSORFLOW_LITE_DELEGATES_NNPAI_JAVA_SRC_MAIN_NATIVE_NNAPI_DELEGATE_JNI_H_
```

Comment:
note a typo: the expected header guard would be 

    TENSORFLOW_LITE_DELEGATES_NNAPI_JAVA_SRC_MAIN_NATIVE_NNAPI_DELEGATE_JNI_H_

Reply:
should be fixed with the line below already?

Reply:
Hmm, still looks wrong? NNPAI -> NNAPI

Reply:
updated. thanks